src/app/components/OrderBook.tsx

```diff
@@ -130,16 +130,31 @@ export function OrderBook() {
   );
   const sells = useAppSelector((state) => state.orderBook.sells);
   const buys = useAppSelector((state) => state.orderBook.buys);
-  //const grouping = useAppSelector((state) => state.orderBook.grouping);
+  const [showOrderBookLabel, setShowOrderBookLabel] = useState(true);
+
+  useEffect(() => {
+    const handleResize = () => {
+      setShowOrderBookLabel(window.innerWidth > 1400);
+    };
+    window.addEventListener("resize", handleResize);
+    handleResize();
+    return () => window.removeEventListener("resize", handleResize);
+  }, []);
+
+  const gridColumnClasses = showOrderBookLabel
+    ? "grid-cols-2"
+    : "grid-cols-1";
 
   return (
     <div className="p-2 text-sx text-primary-content">
-      <div className="grid grid-cols-2 m-1 text-secondary-content font-bold text-sm uppercase">
-        <div className="justify-self-start">{t("order_book")}</div>
+      <div className={`grid m-1 text-secondary-content font-bold text-sm uppercase ${gridColumnClasses}`}>
+        {showOrderBookLabel && (
+          <div className="justify-self-start">{t("order_book")}</div>
+        )}
         <div className="flex justify-end join">
-          <span className="join-item mr-2">{t("grouping")} </span>
+          <span className="join-item mr-2 truncate">{t("grouping")} </span>
```

Comment:
The whole point of using tailwind css is to not to create custom classes (it is difficult to manage them in large projects). Please remove the custom class and use classes provided by tailwind instead. You can search their docs for reference - https://tailwindcss.com/docs

Reply:
Ah, my apologies, I read the PR description incorrectly, everything is good here 👍